styles/index.css

```diff
@@ -43,7 +44,6 @@ main {
   margin: 40px auto;
   display: flex;
   flex-direction: column;
-  align-items: center;
   overflow-y: visible;
   position: relative;
```

Comment:
why do you need the relative property here?

Reply:
because if I don't put position relative here on parent element, the position absolute on child card__bookmark (55) doesn't work.